source/syntax/SyntaxNode.cpp

```diff
@@ -32,7 +32,6 @@ struct GetChildVisitor {
 
     TokenOrSyntax visitInvalid(SyntaxNode&, size_t) { return nullptr; }
 };
-
```

Comment:
This seems like an errant change.

Reply:
Sorry, I don't know whats wrong about this?

Reply:
It's just a random newline deletion in an otherwise unchanged file.